beets/plugins.py

```diff
@@ -291,7 +292,7 @@ def load_plugins(names=()):
             )
 
 
-_instances = {}
+_instances: Dict[Any, Any] = {}
```

Comment:
I think it should be possible to specify accurate types for both the key and value here